src/slick.grid.ts

```diff
@@ -3923,27 +3920,25 @@ export class SlickGrid<TData = any, C extends Column<TData> = Column<TData>, O e
     if ((formatterResult as FormatterResultObject)?.addClasses) {
       addlCssClasses += (addlCssClasses ? ' ' : '') + (formatterResult as FormatterResultObject).addClasses;
     }
-    const toolTip = (formatterResult as FormatterResultObject)?.toolTip ? `title="${(formatterResult as FormatterResultObject).toolTip}"` : '';
 
-    let customAttrStr = '';
+    const toolTipText = (formatterResult as FormatterResultObject)?.toolTip ? `${(formatterResult as FormatterResultObject).toolTip}` : '';
+    const cellDiv = document.createElement('div');
+    cellDiv.className = cellCss + (addlCssClasses ? ' ' + addlCssClasses : '');
+    cellDiv.setAttribute('title', toolTipText);
```

Comment:
you could also use the `createDomElement` util, but in this case you would only save 1 line because util cannot deal with attributes, so you still need to call `setAttribute` separately. So anyway you could skip the use of the util for this one but I would nonetheless prefer to use string literal (with backtick ```) since I find it more readable.
```ts
cellDiv.className = `${cellCss}${addlCssClasses ? ' ' + addlCssClasses : ''}`);
```
maybe another way would be to use the string `trim()` function, that should remove the 2 spaces it adds though it's probably better to verify the output in the browser.
```ts
cellDiv.className = `${cellCss} ${addlCssClasses}`).trim();
```